_posts/0000-01-02-Njoura7.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+add a new file for the practice!
```

Comment:
#I am really enjoying this tutorial and I am already curious  to learn more
![This is an image](https://myoctocat.com/assets/images/base-octocat.svg)